split.Console/Interfaces/IGameInterface.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace split.Console.Interfaces
+{
+    /// <summary>
+    /// This interface is a very rough draft. It will eventually be implemented by a Game
+    /// controller. This interface encapsulates game-centered actions. 
+    /// </summary>
+    public interface IGameInterface
+    {
+        /// <summary>
+        /// This will launch the Game object containing all necessary information
+        /// about the game. It takes a game name as an argument. This will initialize the game. 
+        /// The API call from the front end could be /game/{name}/launch? 
+        /// </summary>
+        /// <returns>An object that implements IGameInterface. </returns>
+        IGameInterface launch(string name);
+
+        /// <summary>
+        /// This is an alternative to the method above. It doesn't return anything. 
+        /// </summary>
+        void launch_alternative(string name);
+
+        /// <summary>
+        /// The API for this could be /game/pot. 
+        /// </summary>
+        /// <returns>Returns the amount that is in the pot. </returns>
+        decimal GetPot();
```

Comment:
Why not just use an integer? Values are unlikely to become very high given that the average game is played with 1, 5, 10, 20 chips. Totally to maybe a max of 1 million "points" in play at a time.